components/card/src/Card.js

```diff
@@ -0,0 +1,64 @@
+import styled from 'react-emotion';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { defaultTheme } from '@verdigris/theme';
+import { ThemeProvider } from 'emotion-theming';
+
+const propTypes = {
+  /**
+   * One of CardHeader, CardBody, CardFooter
+   */
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node),
+  ]),
+  /**
+   * Applies border around entire card when true
+   */
+  hasBorder: PropTypes.bool,
+  /**
+   * Applies shadow below card to simulate 3-d space when true
+   */
+  hasShadow: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: [],
+  hasBorder: false,
+  hasShadow: true,
+};
+
+const Section = styled('section')`
+  ${p => p.theme.Card()(p)}
+`;
+
+function Card (props) {
+  const {
+    children,
+    hasShadow,
+    ...restProps
+  } = props;
+
+  const wrappedChildren = children ? React.Children.toArray(children) : [];
+  const cardHeader = wrappedChildren.filter(child => child.type.displayName === 'CardHeader');
```

Comment:
I do not think this will work out of development mode; which is why I suspect the deployed site is breaking.